personnel.md

```diff
@@ -17,18 +17,18 @@ Feel free to contact the secretaries at info AT php-fig.org. For more informatio
 
 | Name                                  | Term                    |
 |---------------------------------------|-------------------------|
-| Larry Garfield ([@Crell])             | 2016-12-24 - 2021-05-30 |
-| Matthew Weier O'Phinney ([@mwop])     | 2016-12-24 - 2021-05-30 |
+| Larry Garfield ([@Crell])             | 2016-12-24 - 2023-05-30 |
+| Matthew Weier O'Phinney ([@mwop])     | 2016-12-24 - 2023-05-30 |
 | Cees-Jan Kiewiet ([@WyriHaximus])     | 2016-12-24 - 2022-08-31 |
 | Chris Tankersley ([@dragonmantank])   | 2016-12-24 - 2022-01-31 |
 | Korvin Szanto ([@korvinszanto])       | 2016-12-24 - 2022-01-31 |
 | Chuck Burgess ([@ashnazg])            | 2018-08-26 - 2022-08-31 |
-| Woody Gilk ([@shadowhand])            | 2019-05-26 - 2021-05-30 |
-| Matteo Beccati ([@mbeccati])          | 2019-05-26 - 2021-05-30 |
+| Woody Gilk ([@shadowhand])            | 2019-05-26 - 2021-08-31 |
```

Comment:
As we discussed on Discord with @crell, this term should be aligned to the others, and end in 2022, so that we go back to 3 waves of 4 terms:
```suggestion
| Woody Gilk ([@shadowhand])            | 2019-05-26 - 2022-08-31 |
```

Reply:
Good catch, that was indeed my intention, but I forgot to change the year. Also, technically those terms end on the last Sunday of the election month, which is  2022-08-28, but I kept them in line with the current end of term.